src/PrestaShopBundle/Resources/config/routing/admin/configure/shop_parameters/contacts.yml

```diff
@@ -0,0 +1,46 @@
+admin_contacts:
+    path: /
+    methods: GET
+    defaults:
+        _controller: 'PrestaShopBundle:Admin\Configure\ShopParameters\Contacts:index'
+        _legacy_controller: AdminContacts
+
+admin_contacts_search:
+    path: /
+    methods: [POST]
+    defaults:
+        _controller: 'PrestaShopBundle:Admin\Configure\ShopParameters\Contacts:search'
+        _legacy_controller: AdminContacts
+
+
+admin_contacts_create:
```

Comment:
not sure if it should be `create` or `add`.

Reply:
the controller action is called `createAction` as well so I used the same here for consistency. Do you think `add` should be used here?

Reply:
i dont really know which one is more correct here :/ but in BO you click "Add new", not "Create new". Same with "Edit", you click "edit" and not "update", that's why you named it like `admin_contacts_edit` and not `admin_contacts_update`?

Reply:
@matks @PierreRambaud maybe you have some insights about naming?

Reply:
After giving it a thought, I think `create` is slightly better than `add` because:
- it is widely used in web applications
- it is used in CRUD acronym
So developers will understand quickly what a `create.html.twig` or a `createAction` is about.

Reply:
@matks in that case, if we follow CRUD naming convention, we should rename `edit` to `update` as well?

Reply:
I find `update` quite vague 😅  while `create` is really clear about what it does. I find `edit` clear too.

However this is my personal opinion, this is not a very good answer 😅